sdk/python/kfp/_client.py

```diff
@@ -828,14 +828,17 @@ def upload_pipeline_version(
     pipeline_package_path,
     pipeline_version_name: str,
     pipeline_id: Optional[str] = None,
-    pipeline_name: Optional[str] = None
+    pipeline_name: Optional[str] = None,
+    update_default_version: Optional[str] = None,
```

Comment:
nit: use boolean type here is probably a better UX?

Reply:
Yes, makes sens, updated.